domains/gauravkulkarni.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "gkulk007",
+        "email": "gaurav.kulkarni@mescoeorg.onmicrosoft.com"
+      },
+
+      "record": {
+        "CNAME": "www"
```

Comment:
Invalid CNAME

Reply:
updated CNAME

Reply:
Updated the settings and removed the previous domain. Please check once again. Thankyou